packages/styles/scss/patterns/sub-patterns/card/index.scss

```diff
@@ -27,16 +27,29 @@
     }
 
     &__title {
+      margin-bottom: $carbon--spacing-05;
       @include carbon--type-style('expressive-heading-03');
```

Comment:
Should the fluid be set to `true` here?

Reply:
not necessary as 01,02,03 of expressive type are static anyways so won't scale even if I set it to true 

@oliviaflory

Reply:
Correct that expressive heading -01, 02 and 03 are fixed sizes. 

The icon colors should probably be consistent between the two variations, it's possible I had @annawen1  change the color token when we were dealing with the inverse tokens. 

@photodow can I ask a dumb question, what is the difference between the static and link card?

Reply:
> Correct that expressive heading -01, 02 and 03 are fixed sizes.

@oliviaflory sorry got confused because on the website I think they're listed as fluid.

> @photodow can I ask a dumb question, what is the difference between the static and link card?

@oliviaflory static is just a tile none interactible. It has a clickable CTA style text or otherwise  LinkWithIcon. Link is just a clickable tile. I see static possibly coming in handy with the benefits section.